ReactNativeTemplate/installandroid.js

```diff
@@ -4,17 +4,18 @@ var packageJson = require('./package.json')
 var execSync = require('child_process').execSync;
 var path = require('path');
 var fs = require('fs');
-
+var rimraf = require('rimraf');
```

Comment:
The pb is that we don't run npm install until line 21.
So if you start clean (remove node_modules), it will fail.

Reply:
So we should move up the npm install call or move down the rimraf related stuff (or just use --recurse-submodules in the git call).